harper-dictionary-parsing/src/char_string.rs

```diff
@@ -6,6 +6,7 @@ pub type CharString = SmallVec<[char; 12]>;
 
 pub trait CharStringExt {
     fn to_lower(&self) -> CharString;
```

Comment:
Similar to `Span`: I don't think this should be defined in this crate. I think it would perfectly OK to define it in `harper-core` and include `harper-core` as a dependency.

Reply:
The issue is that it gets used in harper-core's build script, so in doing so we create a circular dependency. A new crate would be best, but we already have quite a few.